tests/fixtures/routes/respond-statements.php

```diff
@@ -0,0 +1,4 @@
+
```

Comment:
Why are there two blank lines?

Reply:
https://github.com/laravel-shift/blueprint/blob/db562f297d366e02124a9cc9b6746116aab5faba/src/Generators/RouteGenerator.php#L28

Reply:
Hahaha. 😅

Reply:
I've assumed it was the way you wanted it, after looking at. `routes/readme-example.php`

Reply:
do you want me to fix it?

Reply:
No. That's okay. It's on my list to review the route generation anyway.